api/schema/aws_account_resource.go

```diff
@@ -66,6 +66,33 @@ var (
 						Mode: ImmutableAttributeMode,
 					},
 				},
+				"role_arn": StringResourceAttributeWithMode{
+					StringAttribute: resource_schema.StringAttribute{
+						Description: "Provider AWS role ARN.",
+						Required:    true,
+					},
+					attributeWithMode: attributeWithMode{
+						Mode: WriteOnlyOnceAttributeMode,
+					},
+				},
+				"management_account_id": StringResourceAttributeWithMode{
+					StringAttribute: resource_schema.StringAttribute{
+						Description: "Management account ID.",
+						Optional:    true,
+					},
+					attributeWithMode: attributeWithMode{
+						Mode: WriteOnlyOnceAttributeMode,
+					},
+				},
+				"organization_id": StringResourceAttributeWithMode{
+					StringAttribute: resource_schema.StringAttribute{
+						Description: "AWS organization ID.",
```

Comment:
```suggestion
						Description: "AWS organization ID. If specified, the whole AWS organization is onboarded instead of just the AWS account. If specified, `management_account_id` must also be specified.",
```